test/inductor/test_profiler.py

```diff
@@ -280,6 +281,23 @@ def check_triton_event(e) -> None:
         for e in triton_events:
             check_triton_event(e)
 
+    @unittest.skipIf(not HAS_TRITON, "requires cuda & triton")
```

Comment:
Hi, currently HAS_TRITON is used for verify triton installation. Both CUDA and XPU backend may HAS_TRITON. So This CUDA specific case will break on XPU, I suggest to use `HAS_CUDA`.

Reply:
I believe the test should still pass on XPU, so I'll leave this as is